app/assets/stylesheets/categories.scss

```diff
@@ -0,0 +1,104 @@
+// Place all the styles related to the Categories controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: https://sass-lang.com/
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 10px 0 10px;
+  color: white;
+  background-color: #3778c2;
+  height: 80px;
+}
+
+.header-space {
+  padding-right: 3rem;
+}
+
+.header-title {
+  font-size: 25px;
+  font-weight: 300;
+  font-family: 'Lucida Sans', 'Lucida Sans Regular', 'Lucida Grande', 'Lucida Sans Unicode', Geneva, Verdana, sans-serif;
```

Comment:
- [ ] As per the requirement, the font family should be `Proxima Nova Bold` or `Proxima Nova Light`. Please update to follow the font face requirement.